public/javascripts/govuk/details.polyfill.js

```diff
@@ -6,188 +6,185 @@
 
```

Comment:
I think we need to update this script tag to reference the new location: https://github.com/alphagov/govuk_elements/blob/32553af219c7b9fbad74aa71feebb32f38bcf643/app/views/includes/scripts.html#L9

It's 404ing when I run this new updated branch locally.